Correct max request length in HTTP bindings trigger docs

snoopdouglas opened the pull request:
This value was incorrect; you'll note that in the docs for `<httpRuntime>`, `maxRequestLength` is specified in kB while the other values are in bytes (or 'characters'):

https://msdn.microsoft.com/en-us/library/e1f13641(v=vs.100).aspx

azurecla commented:
Hi __@ciphertxt__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@ciphertxt - Thank you, merging!

snoopdouglas committed:
Correct max request length in HTTP bindings trigger docs

This value was incorrect; you'll note that in the docs for `<httpRuntime>`, `maxRequestLength` is specified in kB while the other values are in bytes (or 'characters'):

https://msdn.microsoft.com/en-us/library/e1f13641(v=vs.100).aspx

snoopdouglas committed:
Uppercase 'K' in KB
